i18n/en/code.json

```
@@ -8,52 +8,52 @@
     "description": "homepage install button"
   },
   "home.quickstart": {
-    "message": "All About Your Modern Apps by Platform Engineering"
+    "message": "Everything You Need to Know About Modern Apps with Platform Engineering."
   },
   "home.easyshipping": {
     "message": "Dynamic Configuration Management"
   },
   "home.easyshipping.1": {
-    "message": "Manage all application operations in one place, in a unified easy way"
+    "message": "Easily manage all application operations from a unified platform."
   },
   "home.easyshipping.2": {
-    "message": "Environment-agnostic application configurations"
+    "message": "Environment-agnostic application configuration for seamless deployment."
   },
   "home.easyshipping.3": {
-    "message": "Standardized and flexible platform configurations"
+    "message": "Standardized yet flexible platform configurations tailored to your needs."
```

```suggestion
    "message": "Standardized yet flexible platform configurations."
```
Same reason as above.

---

\This change was made with the same thought process in mind. However, yes, we can make it shorter to keep the UI tight. But keep in mind, a shorter version may come across as more generic rather than audience-driven, as I mentioned in my previous comment.

---

OK